config/locales/devise.en.yml

```diff
@@ -6,10 +6,12 @@ en:
       confirmed: "Your email address has been successfully confirmed."
       confirmed: |
         Thank you, your email address is now verified.
-
+        <br/>
```

Comment:
I might be wrong but is there something about not using multiple ```<br/>``` tags, for accessibility?

Reply:
It is not semantic HTML